ast/versions.ml

```diff
@@ -515,3 +532,20 @@ include Register_migration(OCaml_411)(OCaml_412)
 (*$*)
 
 module OCaml_current = OCaml_OCAML_VERSION
+
+module Find_version = struct
+  type t = Impl of (module OCaml_version) | Intf of (module OCaml_version) | None
```

Comment:
While naming a constructor `None` is a valid thing to do, the `option` type is so pervasives that it seems best to not reuse the `None` constructor. In this case, `Uknown` seems like a good name (and even better IMO):

```suggestion
  type t = Impl of (module OCaml_version) | Intf of (module OCaml_version) | Unknown
```

Reply:
Sounds good, thanks!